<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-U4D6HB2" failures="2" tests="9" name="TestFaildCaseSenario" time="0.101" errors="0" timestamp="2022-03-26T00:02:40 IST">
  <testcase classname="testNGPracice.TestSuite1" name="t1" time="0.01"/>
  <testcase classname="testNGPracice.TestSuite1" name="t2" time="0.005">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at testNGPracice.TestSuite1.t2(TestSuite1.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- t2 -->
  <testcase classname="testNGPracice.TestSuite1" name="t3" time="0.001"/>
  <testcase classname="testNGPracice.TestSuite1" name="t4" time="0.001">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at testNGPracice.TestSuite1.t4(TestSuite1.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- t4 -->
  <testcase classname="testNGPracice.TestSuite1" name="t5" time="0.002"/>
  <testcase classname="testNGPracice.TestSuite1" name="t6" time="0.002"/>
  <testcase classname="testNGPracice.TestSuite1" name="t7" time="0.002"/>
  <testcase classname="testNGPracice.TestSuite1" name="t8" time="0.001"/>
  <testcase classname="testNGPracice.TestSuite1" name="t9" time="0.001"/>
</testsuite> <!-- TestFaildCaseSenario -->
